Sources/ClientRuntime/Networking/Http/HttpContext.swift

```diff
@@ -1,6 +1,8 @@
 // Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
Changes in this file are either of the two below:
- Completely new setter, getter, or attribute key for new context properties
- Setter, getter, or attribute key moved to / from aws-sdk-swift and deleted from / added to here

At the end of the file, the attribute keys have been refactored to be in one place for code organization.

Reply:
Clarifying question: this makes sense, but is it ok to expose all of these keys on `HttpContext`, or is there a way to keep them more private?

Reply:
The reason for exposing these attribute keys as public is because we need to use these in aws-sdk-swift, which is a separate module that imports ClientRuntime. There does exist a way to keep them more private by "hiding" these by using a @_spi notation, but it is not finalized and still an experimental feature Swift offers.